File: TeeNew/ChartGrid_Colors.pas
unit ChartGrid_Colors;
{$I TeeDefs.inc}

interface

uses
  {$IFNDEF LINUX}
  Windows, Messages,
  {$ENDIF}
  SysUtils, Classes,
  {$IFDEF CLX}
  QGraphics, QControls, QForms, QDialogs, QExtCtrls, QStdCtrls, QComCtrls,
  QGrids,
  {$ELSE}
  Graphics, Controls, Forms, Dialogs, ExtCtrls, StdCtrls, ComCtrls,
  Grids,
  {$ENDIF}
  Chart_Grid, TeeChartGrid, TeeNavigator, TeCanvas, TeEngine, Series,
  TeeProcs, Chart;

type
  TChartGridColors = class(TChartGridForm)
    CheckBox3: TCheckBox;
    CheckBox4: TCheckBox;
    procedure CheckBox3Click(Sender: TObject);
    procedure CheckBox4Click(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

implementation

{$IFNDEF CLX}
{$R *.DFM}
{$ELSE}
{$R *.xfm}
{$ENDIF}

procedure TChartGridColors.CheckBox3Click(Sender: TObject);
begin
  ChartGrid1.ShowColors:=CheckBox3.Checked;
end;

procedure TChartGridColors.CheckBox4Click(Sender: TObject);
begin
  ChartGrid1.ShowFields:=CheckBox4.Checked;
end;

initialization
  RegisterClass(TChartGridColors);
end.

File: TeeNew/Builder2009/ChartGrid_Colors.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "ChartGrid_Colors.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "Chart_Grid"
#pragma link "TeeChartGrid"
#pragma link "TeeNavigator"
#pragma resource "*.dfm"
TChartGridColors *ChartGridColors;
//---------------------------------------------------------------------------
__fastcall TChartGridColors::TChartGridColors(TComponent* Owner)
        : TChartGridForm(Owner)
{
}
//---------------------------------------------------------------------------

void __fastcall TChartGridColors::CheckBox3Click(TObject *Sender)
{
  ChartGrid1->ShowColors=CheckBox3->Checked;
}
//---------------------------------------------------------------------------

void __fastcall TChartGridColors::CheckBox4Click(TObject *Sender)
{
  ChartGrid1->ShowFields=CheckBox4->Checked;
}
//---------------------------------------------------------------------------

